contracts/root/RootChain.sol

```diff
@@ -16,38 +16,19 @@ contract RootChain is RootChainStorage, IRootChain {
     using RLPReader for RLPReader.RLPItem;
 
     modifier onlyDepositManager() {
-        require(
-            msg.sender == registry.getDepositManagerAddress(),
-            "UNAUTHORIZED_DEPOSIT_MANAGER_ONLY"
-        );
+        require(msg.sender == registry.getDepositManagerAddress(), "UNAUTHORIZED_DEPOSIT_MANAGER_ONLY");
         _;
     }
 
-    function submitHeaderBlock(bytes calldata data, bytes calldata sigs)
-        external
-    {
-        (
-            address proposer,
-            uint256 start,
-            uint256 end,
-            bytes32 rootHash,
-            bytes32 accountHash,
-            uint256 _borChainID
-        ) = abi.decode(
-            data,
```

Comment:
this looks more readable

Reply:
this is due to max length of the line. We can change it in the config.
:D bad idea for this PR.